src/Avalonia.X11/X11Window.cs

```diff
@@ -400,7 +401,8 @@ ev.ButtonEvent.button switch
                             2 => RawPointerEventType.MiddleButtonDown,
                             3 => RawPointerEventType.RightButtonDown,
                             8 => RawPointerEventType.XButton1Down,
-                            9 => RawPointerEventType.XButton2Down
+                            9 => RawPointerEventType.XButton2Down,
+                            _ => throw new NotImplementedException()
```

Comment:
> why not?

because throwing an exception says it no longer valid to have other values, and thats different from the current behavior... where it just ignores anything else.

Reply:
`switch` will throw an exception anyway, just a different type. We should probably refactor the code to ignore the event entirely